### ISSUE TITLE
is this piracy or no?

### ISSUE THREAD

**Issue by ghost** [Pixelsuft/hl#7]

**Comment by tetonis** [comment]
No. You have to provide your own copies "valve" folder and the day one zips that come with are fair game.

**Comment by ghost** [comment]
what about opposing forces? I didn't know there was a day one of those @tetonis

**Comment by tetonis** [comment]
Your not going to get in trouble especially if you own half-life anyway. Also this project is dead as hell.

**Comment by StonkMan21** [comment]
Do you know where I could "get" the game folder used for this, I want to get mods running

**Comment by Xanderplayz16-SCHOOL** [comment]
You just go to halva_en_out, then combine the files together into one, open it and install the mods, then when you are done, split into 10MiB chunks with the same naming format and overwrite the old ones. Then run a webserver (make sure to apply the SharedArrayBuffer cors flags!) and then boom! Half-Life in the browser!
